Project2015To2017/Reading/Conditionals/FunctionCallExpressionNode.cs

```diff
@@ -12,11 +12,11 @@ namespace Project2015To2017.Reading.Conditionals
     internal sealed class FunctionCallExpressionNode : OperatorExpressionNode
     {
         private readonly List<GenericExpressionNode> _arguments;
-        private readonly string _functionName;
+        public readonly string FunctionName;
```

Comment:
Not that I like changing MSBuild source, but I believe the change is small enough and presents better codestyle.